lib/commands/fastboot.js

```diff
@@ -64,9 +64,8 @@ module.exports = {
     this.commandOptions = options;
 
     var runCommand = function() {
-      var appName = process.env.EMBER_CLI_FASTBOOT_APP_NAME || this.project.name();
-
-      return this.runCommand(appName, options);
+      var appOutputPaths = JSON.parse(process.env.EMBER_CLI_FASTBOOT_APP_OUTPUT_PATHS);
```

Comment:
@rwjblue not sure how to proceed here. Since the app has to be built at least once to serve it, write the output paths to a temp file?